lifecycle-operator/apis/lifecycle/v1beta1/keptnappversion_types.go

```diff
@@ -40,6 +40,9 @@ type KeptnAppVersionSpec struct {
 	// TraceId contains the OpenTelemetry trace ID.
 	// +optional
 	TraceId map[string]string `json:"traceId,omitempty"`
+	// +optional
+	// Metadata contains additional key-value pairs for contextual information.
+	Metadata map[string]string `json:"metadata,omitempty"`
```

Comment:
Let's not copy the parameters from one struct to another, but let's instead use the KeptnAppContextSpec here. This needs a little bit of refactoring, which was mostly done by https://github.com/keptn/lifecycle-toolkit/pull/2833

Reply:
done 👍